java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/AppetizerOrdered.java

```diff
@@ -0,0 +1,13 @@
+package com.my.FoodTruckApp.order;
```

Comment:
I think this belongs in the  appetizer directory

Reply:
This has been moved